src/knock.ts

```diff
@@ -77,7 +77,7 @@ class Knock {
    * @param options Optionally specify the signing key to use (in PEM or base-64 encoded format), and how long the token should be valid for in seconds
    * @returns {string} A JWT token that can be used to authenticate requests to the Knock API (e.g. by passing into the <KnockFeedProvider /> component)
    */
-  static signUserToken(userId: string, options: SignUserTokenOptions) {
+  static async signUserToken(userId: string, options: SignUserTokenOptions) {
```

Comment:
Moving `signUserToken` from a sync to async function is the only breaking change I can see here. Should we update the JSDoc return type to be a promise to make this more clear? `@returns {Promise<string>}` or maybe consider using the `@async` tag: https://jsdoc.app/tags-async.

Reply:
Good call out, I'll update the return type